macaroonbakery/checkers/declared.py

```diff
@@ -3,7 +3,7 @@
 
 from macaroonbakery.checkers.namespace import Namespace
 from macaroonbakery.checkers.caveat import parse_caveat
-from macaroonbakery.checkers import COND_DECLARED, STD_NAMESPACE
+from macaroonbakery.checkers.conditions import COND_DECLARED, STD_NAMESPACE
 from macaroonbakery.auth_context import ContextKey
```

Comment:
I don't think that checkers should be importing from bakery (gives high likelihood of import cycles).
Maybe that's an argument for moving AuthContext into checkers.

Reply:
yep